test/integration/transformation_filter_integration_test.cc

```diff
@@ -475,12 +475,13 @@ TEST_P(TransformationFilterIntegrationTest, BodyHeaderTransform) {
   EXPECT_EQ(expected_request, actual_request);
 
   json actual_response = json::parse(response->body());
+  // remove the `x-envoy-upstream-service-time` as it is not deterministic.
```

Comment:
can we be a bit more specific here? e.g. header order rendering is not deterministic

Reply:
https://github.com/solo-io/envoy-gloo/pull/209/commits/724f11cd3ba5b8a2849e120197e7ed97bad12373